### ISSUE TITLE
Executable Path is a Directory

### ISSUE THREAD

**Issue by AwayCoast23465** [ckosmic/sm64ex-ios#33]
**Describe the bug**

When I attempt to run the app for ios, it says Executable Path is a Directory: /Users/<my_name>/Library/Developer/Xcode/DerivedData/sm64ios-adtogoxpdmdahwdnxmmdissyqagw/Build/Products/Debug-iphoneos/sm64ios.app is not a valid path to an executable file. 

(also, im new to this, so can you explain it in layman's terms?)

**To Reproduce**

follow all instructions and when i try to run, see the error

run on my iphone 8 plus

**Screenshots**

<img width="1387" alt="Screenshot 2023-02-03 at 5 05 02 pm" src="https://user-images.githubusercontent.com/113707774/216525077-23fc13d1-ccd2-4da4-99ac-fda2c57a5b8e.png">

**Desktop (please complete the following information):**
 - OS: MacOS 13.1 Ventura
 - Browser Chrome

**Smartphone (please complete the following information):**
 - Device: Iphone 8 plus
 - OS: iOS16.0.2
 - Browser [Oprea]

**Comment by SlickAmogus** [comment]
Same here. Followed steps exactly several times. Nothing online seems relevant. The build is succeeding.

**Comment by nogueira-r** [comment]
Have you encountered any fix for this? I'm getting the same problem as you

**Comment by ckosmic** [comment]
Are you building on an M1 or M2 Mac? I just created a branch that fixes compilation errors on Apple silicon if you'd like to check it out: https://github.com/ckosmic/sm64ex-ios/tree/apple_silicon_fix
